roles/percona-server/tasks/serverspec.yml

```diff
@@ -8,3 +8,5 @@
             dest=/etc/serverspec/spec/localhost/
             mode=0755
   with_fileglob: ../templates/etc/serverspec/*
+  notify:
+    - restart mysql server
```

Comment:
Let's remove restart mysql from here as this is only needed one time for 3.1.1 update. Probably good to create a separate playbook fix-mysql-gcache.yml